src/rgw/rgw_swift_auth.h

```diff
@@ -142,6 +144,16 @@ class ExternalTokenEngine : public rgw::auth::Engine {
   }
 };
 
+/* SwiftAnonymous: applier. */
+class SwiftAnonymousApplier : public rgw::auth::LocalApplier {
+  public:
+    SwiftAnonymousApplier(CephContext* const cct,
+                          const RGWUserInfo& user_info)
+      : LocalApplier(cct, user_info, LocalApplier::NO_SUBUSER, boost::none) {
+      };
+    bool is_admin_of(const rgw_user& uid) const {return false;}
+    bool is_owner_of(const rgw_user& uid) const {return false;}
+};
```

Comment:
This isn't strictly required, but might be nice to say an anonomous user is never an owner or admin?